janusgraph-core/pom.xml

```diff
@@ -116,15 +116,15 @@
         </testResources>
         <plugins>
             <plugin>
-                <groupId>org.codehaus.gmaven</groupId>
-                <artifactId>gmaven-plugin</artifactId>
+                <groupId>org.codehaus.gmavenplus</groupId>
+                <artifactId>gmavenplus-plugin</artifactId>
                 <version>1.5</version>
                 <executions>
                     <execution>
                         <goals>
                             <goal>generateStubs</goal>
                             <goal>compile</goal>
-                            <goal>generateTestStubs</goal>
+                            <goal>testGenerateStubs</goal>
```

Comment:
This goal was renamed in this plugin.  But, it turns out there's no groovy sources at all in the janusgraph-core project, so I'm not sure what this is supposed to be doing.